tests/Functional/Platform/NewPrimaryKeyWithNewAutoIncrementColumnTest.php

```diff
@@ -47,7 +47,7 @@ public function testAlterPrimaryKeyToAutoIncrementColumn(): void
         $newTable->dropPrimaryKey();
         $newTable->setPrimaryKey(['new_id']);
 
-        $diff = (new Comparator())->compare($schema, $newSchema);
+        $diff = (new Comparator())->compareSchemas($schema, $newSchema);
```

Comment:
The tests should not change unless it's a bug or a breaking change. This way, we can make sure that the deprecated API still functions properly.